test/BUILD.bazel

```diff
@@ -82,6 +82,8 @@ cc_test(
         "unpack.cc",
     ],
     copts = copts(),
+    # Use a faster implementation of malloc (and show that tests pass with it).
+    malloc = "@com_github_jemalloc_jemalloc//:jemalloc",
```

Comment:
```suggestion
    select({
        # NOTE!
        # For now we build jemalloc on macOS and Linux.
        # The way how to build jemalloc on Windows is more
        # complicated. In future we are going to make this
        # lib build on Windows too.
        # https://github.com/3rdparty/bazel-rules-jemalloc/tree/ArthurBandaryk.jemalloc-windows
        "@bazel_tools//src/conditions:windows": [],
        "//conditions:default": [
            "@com_github_jemalloc_jemalloc//:jemalloc",
        ],
```

Reply:
This won't work as-is since `malloc` needs to be a single entry, not a list.

My first attempts at getting `malloc` working with a `select` statement failed, but I'll give it another go.